spec/models/mechanic_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "rails_helper"
+
+RSpec.describe Ride, type: :model do
+  describe "relationships" do
+    it { should have_many(:rides) }
+  end
```

Comment:
If we take a look at the `Ride` model you have, there is one association on it currently:

```ruby
class Ride < ApplicationRecord
  belongs_to :amusement_park
end
```

The association you have here in your tests doesn't match what's in the model. This test should look like so:

```ruby
describe "relationships" do
  it { should belong_to(:amusement_park) }
end
```

We have a `belongs_to` in our model, so we need a `should belong_to` test.